src/singleton.ts

```
@@ -0,0 +1,45 @@
+        return this.meetingParams
+    }
+
+    public isServerless(): boolean {
+        if (this.meetingParams === null) {
+            throw new Error('Meeting params are not set')
+        }
+        return this.meetingParams.remote === null
+    }
+}
+
+export let GLOBAL = new Global()
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'GLOBAL' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)